bundle/manifests/sandboxed-containers-operator.clusterserviceversion.yaml

```diff
@@ -399,6 +399,8 @@ spec:
                   value: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-builder-rhel9@sha256:fec854750f7c5e137d0f77d2069e8a8cb66b17d6e4fc8f8eec2df22b56b2772e
                 - name: RELATED_IMAGE_PODVM_PAYLOAD
                   value: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-payload-rhel9@sha256:34ae87b5563b01046045baa0c34f609297e6a7811e87bbca1287847e6257e13a
+                - name: RELATED_IMAGE_PODVM_OCI
+                  value: registry.redhat.io/openshift-sandboxed-containers/osc-dm-verity-image@sha256:4959a0afc5689cfc846d6731f595e57b76b5638c1f40d2f4d9c5642640e771fe
```

Comment:
All our images (except the bunlde) have "rhel9" in their names. Maybe this one should too?